### ISSUE TITLE
Rendering zero categories.

### ISSUE THREAD

**Issue by connorjclark** [GoogleChrome/lighthouse#8881]
Do the wrong thing:

```
node lighthouse-cli https://www.nyt.com --view --only-categories performance,seo
```

See an empty report and fireworks :)

![image](https://user-images.githubusercontent.com/4071474/57244696-d1d76f80-6fed-11e9-9a4d-d5fbc67d6912.png)

**Comment by patrickhulce** [comment]
Somewhat expected :) there's no category called `performance,seo`, yargs does array types like `--only-categories=performance --only-categories=seo` 

we could split on `,` and flatten though, it wouldn't be too bad

**Comment by brendankenny** [comment]
fixed by a combo of #8933 and #9073
